spec/integration/watched_words_spec.rb

```diff
@@ -64,6 +64,14 @@ def should_block_post(manager)
       should_block_post(manager)
     end
 
+    it "look at title too" do
+      block_word = Fabricate(:watched_word, action: WatchedWord.actions[:block], word: "abc")
+      manager =
+        NewPostManager.new(tl2_user, title: "Hello world", raw: "abcódef", topic_id: topic.id)
+
+      expect(manager.perform).to be_success
```

Comment:
No, this is correct. It used to block `abcódef` because it considered it as two words `abc def` as it treated non-alphanumeric characters as word boundaries.